### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by adamsdenniskariuki**
Review my code for create_room and add_person. Thanks.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+class Room(object):
+	
+	def __init__(self, room_name, room_type):
+		self.created_rooms = {}
+		self.room_name = room_name
+		self.room_type = room_type
+
+	def create_room(self, room_name, room_type):
+		
+		if(isinstance(self.room_name, list) == True):
```
Comment (ghost) [inline app.py]: Redundant statement. 

> isinstance

 returns True or False already.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+	def __init__(self):
+		self.created_fellow = []
+
+	def add_person(self, person_name, person_type, wants_accommodation = 'N'):
+		super(Fellow, self).__init__(person_name, person_type, wants_accommodation)
+		self.created_fellow = super(Fellow, self).add_person(self.person_name, 
+			self.person_type, self.wants_accommodation)
+		return self.created_fellow
+		
+import cmd
```
Comment (ghost) [inline app.py]: Import statements should ideally appear at the top of your file, before any class definition, unless there is a condtitional execution that needs at most an extra module that is not required anywhere else.

**Review by ghost — COMMENTED** [review]
Consider a more descriptive filename. app.py is very generic.

File: .cache/v/cache/lastfailed [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (morriskimani) [inline .cache/v/cache/lastfailed:1]: Please gitignore this file. Gitignore the whole `.cache` folder.

File: .coverage [hunk 4]
```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines":{"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\tests\\test_livingspace.py":[1,2,5,33,8,9,12,13,14,15,16,17,20,21,22,25,26,29,30],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\tests\\test_office.py":[32,1,2,5,8,9,12,13,14,15,16,17,20,21,22,25,26,29,30],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\model\\room.py":[2,4,5,6,7,9,11,16,17,21,23,24,25,27,28,29,30,31,32,33,37,39,40,41,43,44,45,46,47,48,49],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\tests\\test_fellow.py":[1,2,5,8,9,12,13,14,15,16,17,18,21,22,23,26,27,30],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\tests\\test_dojo.py":[32,1,2,33,36,5,7,8,11,12,13,16,17,18,21,22,23,26,27,28,31],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\model\\person.py":[2,4,5,6,7,8,10,12,13,15,16,17,18,20,21,23,27,29,30,32,33,34,35,36,37,41,43,44,46,47,48,49,50,51],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\app.py":[1,2,3,4,5,6,10,11,12,13,14,15,18,67,21,32,33,169,42,44,301,270,307,311,56,65,195,69,72,55,76,77,78,142,215,94,96,98,101,105,106,107,242,123,125,126],"E:\\Dojo\\v2\\Adams-Kariuki-Dojo-Project\\tests\\test_staff.py":[1,2,5,8,9,12,13,14,15,16,17,18,21,22,23,26,27,30]}}
```
Comment (morriskimani) [inline .coverage:1]: gitignore this one too.

File: app.py [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+    	if(isinstance(room_name, list)):
+    		if(self.created_rooms):
+    			for r in room_name:
+    				if(r in self.created_rooms):
+    					print("Room already exists!")
+    					return
+    		for room in room_name:
+    			if(not room.isalpha()):
+    				print("Use alphabet (a-z) characters for the room name and room type")
+    				return
```
Comment (morriskimani) [inline app.py:30]: Small flaw in your logic. If a user enters just one wrong `room name`, the whole operation fails.

**Pull request closed without merging**